routemaster/lua/nack.lua

```diff
@@ -0,0 +1,12 @@
+--
+-- Negative-acknowledges a message.
+--
+-- KEYS[1]: list of new message UUIDs
+-- KEYS[2]: zset of pending (nacked) message UUIDs
```

Comment:
If that's confusing — I could pass the same 3 keys to all the scripts.
Do you think that'd help? The performance loss would be tiny.